examples/user_profile_examples/public/avatar_demo.tsx

```diff
@@ -13,6 +13,7 @@ import { PanelWithCodeBlock } from './panel_with_code_block';
 export const AvatarDemo: FunctionComponent = () => {
   const userProfile = {
     uid: 'u_J41Oh6L9ki-Vo2tOogS8WRTENzhHurGtRc87NgEAlkc_0',
+    enabled: true,
```

Comment:
nit: might be worth typing these so that we get type errors before test failures 🤷‍♂️

Reply:
Yep, it's a good idea, will do!